resources/assets/less/bem/beatmap-pack.less

```diff
@@ -51,11 +52,24 @@
     font-size: 90%;
   }
 
+  &__clear {
```

Comment:
Make it modifier of `__cell`. Also for others which for some reason are two elements (of same block) at same time.